packages/block-editor/src/hooks/line-height.js

```diff
@@ -38,6 +38,8 @@ export function LineHeightEdit( props ) {
 	};
 	return (
 		<LineHeightControl
+			__unstableInputWidth="100%"
+			__nextHasNoMarginBottom={ true }
```

Comment:
Unrelated to this PR, but just a though re. the deprecation strategy: how do we avoid that the `__nextHasNoMarginBottom` props sticks around after the migration is complete? Should we add a warning when that prop is passed to the component?

Reply:
I'm hoping we can just silently ignore it, because a void prop is harmless and it kind of punishes early adopters by adding a bunch of console noise they have to deal with again.

For consumers that do care about meaningless props sticking around in their code, I think we can give them enough signal with:

- A note in the readme, as @aaronrobertshaw [suggested](https://github.com/WordPress/gutenberg/pull/37160#discussion_r805471685).
- A changelog entry when the prop is fully removed.
- If both our component and the consumer is TypeScript-ed, that should flag any invalid props. (Another reason to push for TypeScript migrations!)

Does that sound reasonable?